src/ontology/mondo-edit.obo

```diff
@@ -223220,22 +223220,21 @@ property_value: IAO:0000589 "5-oxoprolinase deficiency (disease)" xsd:string
 
 [Term]
 id: MONDO:0009826
-name: PA polymorphism of alpha-2-globulin
+name: obsolete PA polymorphism of alpha-2-globulin
 comment: Reason of obsoletion: out of scope - MONDO:excludePhenotype. Term to consider: -
-subset: obsoletion_candidate
 synonym: "alpha-2-globulin polymorphism PA" EXACT [OMIM:260100]
 synonym: "PA polymorphism of alpha-2-globulin" EXACT [OMIM:260100]
 synonym: "XH antigen" EXACT [OMIM:314800]
-xref: MEDGEN:111766 {source="MONDO:equivalentTo", source="MONDO:MEDGEN"}
-xref: MESH:C009691 {source="MONDO:equivalentTo"}
-xref: OMIM:260100 {source="MONDO:equivalentTo"}
-xref: OMIM:314800 {source="MONDO:equivalentObsolete"}
-xref: UMLS:C0604085 {source="MONDO:equivalentTo", source="MONDO:MEDGEN", source="MEDGEN:111766"}
-is_a: MONDO:0003847 {source="https://orcid.org/0000-0001-5208-3432"} ! hereditary disease
+xref: MEDGEN:111766 {source="MONDO:obsoleteEquivalent", source="MONDO:MEDGEN"}
+xref: MESH:C009691 {source="MONDO:obsoleteEquivalent"}
+xref: OMIM:260100 {source="MONDO:obsoleteEquivalent"}
+xref: OMIM:314800 {source="MONDO:obsoleteEquivalent"}
```

Comment:
This one should be "obsolete equivalent obsolete" (because it is obsolete in Mondo and also obsolete in OMIM)